protocol/contracts/interfaces/oud/IOudRedeemer.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity ^0.8.17;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Oud (protocol/contracts/interfaces/oud/IOudtoken.sol)
+
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import {IERC20Permit} from '@openzeppelin/contracts/token/ERC20/extensions/draft-IERC20Permit.sol';
+
+/**
+ * @title Allows for the redemption of Oud + a stable coin for Temple at the Temple Treasury Price index
+ */
+interface IOudRedeemer {
+  //  @notice Returns the index at which 'stable' token is required to mint Temple
```

Comment:
To be proper natspec, needs to have three `///` (or use the `/**` notation like you have below)

Reply:
fixed in [a887d99](https://github.com/TempleDAO/temple/pull/760/commits/a887d990cd0aa465cc7c5237e60d4e7a1f580dc1)